mitxpro/templates/500.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% load static wagtailcore_tags wagtailimages_tags wagtailmetadata_tags render_bundle %}
```

Comment:
I don't think we want to add `wagtailmetadata_tags` here.

Reply:
Removed